src/python/pants/core/util_rules/external_tool.py

```diff
@@ -121,6 +157,20 @@ def register_options(cls, register):
             help=help_str,
         )
 
+        register(
+            "--use-unsupported-version",
+            advanced=True,
+            type=UnsupportedVersionUsage,
+            help=textwrap.dedent(
+                "    "
+                + UnsupportedVersionUsage.__doc__.format(
+                    tool=cls.name,
+                    constraints=cls.version_constraints if cls.version_constraints else "any",
+                )
+            ),
```

Comment:
Imo, it's better to remove the docstring from the enum and put it directly here. This is harder to follow what's going on, and easier for us to mess up like someone changing the docstring without realizing the coupling.